ldelf/ta_elf.c

```diff
@@ -149,7 +149,30 @@ static TEE_Result e64_parse_ehdr(struct ta_elf *elf, Elf64_Ehdr *ehdr)
 
 	return TEE_SUCCESS;
 }
-#else /*ARM64*/
+#elif defined RV64
```

Comment:
`#elif defined(RV64)`

You should probably add some `#if defined(ARM32) || defined(ARM64)` above `e32_parse_ehdr()` and provide a Risc-V alternative for that too.